sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -150,6 +150,58 @@ object IntervalUtils {
     }
   }
 
+  private val unquotedDaySecondPattern =
+    "([+|-])?(\\d+) (\\d{1,2}):(\\d{1,2}):(\\d{1,2})(\\.\\d{1,9})?"
+  private val quotedDaySecondPattern = (s"^$unquotedDaySecondPattern$$").r
```

Comment:
`unquotedDaySecondRegex`?

Reply:
> `unquotedDaySecondRegex`?

Change this in https://github.com/apache/spark/pull/32444